src/test/java/io/ddavison/conductor/FrameworkTest.java

```diff
@@ -26,6 +26,11 @@ public void testSetText() throws Exception {
     }
 
     @Test
+    public void testSetAndValidateText() throws Exception {
+        setAndValidateText("#setTextField", "set and validate text test");
```

Comment:
I'll be super glad to merge this, as soon as you just add the 

`.validateText(...)` bit in the test.  It's redundant, I agree, but it's visibly obvious that it will validate correctly from the test